less/components/header.less

```diff
@@ -1,4 +1,15 @@
-.header {
+.page-header {
+    padding: 0 1em;
+    color: #FFFFFF;
```

Comment:
I'm very strongly against variables that are color names, so not `@white`. And if it's not a variable, I think it really doesn't matter, especially for white (or black).